docs/redis.md

```diff
@@ -78,6 +86,8 @@ To configure a receiver for your redis logs, specify the following fields:
 | `exclude_paths`       | `[]`                          | A list of filesystem path patterns to exclude from the set matched by `include_paths`.
 
 
+
+
 Example Configuration:
 
 ```yaml
```

Comment:
Looks like there are some `submodules/fluent-bit` changes that accidentally got in. Might need rebasing?

Reply:
Fixed